docs/core/logging.md

```diff
@@ -563,35 +444,35 @@ we provide [built-in JMESPath expressions](#built-in-correlation-id-expressions)
 
 #### Custom keys
 
-???+ warning "Custom keys are persisted across warm invocations"
-    Always set additional keys as part of your handler method to ensure they have the latest value, or explicitly clear them with [`clearState=true`](#clearing-state).
+** Using StructuredArguments **
```

Comment:
"Appending Additional Keys" ? So that the heading reflects when you can learn to do, not a detail of how to do it